htdocs/theme/eldy/ckeditor/config.js

```diff
@@ -32,6 +32,7 @@ CKEDITOR.editorConfig = function( config )
 	//config.autoParagraph = false;
 	//config.removeFormatTags = 'b,big,code,del,dfn,em,font,i,ins,kbd';		// See also rules on this.dataProcessor.writer.setRules
 	//config.forcePasteAsPlainText = true;
+	config.entities_latin = false;
```

Comment:
A better choice would be to add it into doleditor.class.php after line
/* property:xxx is same than CKEDITOR.config.property = xxx */
so we can set the value to true of flase depending if $conf->db->dolibarr_main_db_collationcontains the key 'latin'or not.
Can you try this ?

Reply:
I'm not sure it depends on the database collation
the screen charset is I think always utf-8 
`<meta charset="utf-8">`
I tested on 2 instances
the first dolibarr 16 
`$dolibarr_main_db_character_set='latin1';`
`$dolibarr_main_db_collation='latin1_swedish_ci';`

the second dolibarr 21 alpha
`$dolibarr_main_db_character_set='utf8';`
`$dolibarr_main_db_collation='utf8_unicode_ci';`

In both cases, 
`config.entities_latin = false`
Is ok.

I think in ckeditor, _config.entities_latin_ is not the right word, It should be _config.html_entities_